packages/s2-core/src/interaction/event-controller.ts

```diff
@@ -188,8 +188,16 @@ export class EventController {
     interaction.reset();
   }
 
+  private isMouseEvent(event: Event): event is MouseEvent {
```

Comment:
我是意思是这样, 本身就是一个 boolean 值, 为啥还要单独 return `true` 和 `false`

```suggestion
  private isMouseEvent(event: Event): event is MouseEvent {
   return hasIn(event, 'clientX') && hasIn(event, 'clientY')
  }
```